pytest_rts/utils/common.py

```diff
@@ -65,42 +68,41 @@ def tests_from_changed_srcfiles(diff_dict, files, db_helper):
 
         if any(
             [
-                not db_helper.mapping_line_exists(file_id, line_id)
-                for line_id in new_lines
+                not mappinghelper.line_exists(file_id, line_number)
```

Comment:
just `if any(not ... )`, not `if any([not ...])`. In the first case you'll get a generator inside, and it'll iterate over it only as long as needed to decide that `any` is True. In the second case the complete list is built first.

Reply:
Also you have kind of double negation here; replace `any(not blah)` with `all(blah)`.

Reply:
I also think `not all(blah)` is more readable than `any(not blah)`.